icons/beatport.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Beatport</title><path d="M14.668 24c-3.857 0-6.935-3.039-6.935-6.974a6.98 6.98 0 011.812-4.714l-4.714 4.714-2.474-2.474 5.319-5.26c.72-.72 1.09-1.656 1.09-2.688V0h3.487v6.604c0 2.026-.72 3.74-2.123 5.143l-.156.156a6.945 6.945 0 014.694-1.812c3.955 0 6.975 3.136 6.975 6.935A6.943 6.943 0 0114.668 24zm0-10.714c-2.123 0-3.779 1.753-3.779 3.74 0 2.045 1.675 3.78 3.78 3.78a3.804 3.804 0 003.818-3.78c0-2.065-1.715-3.74-3.819-3.74Z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Beatport</title><path d="M21.641 17.026A6.94 6.94 0 0 1 14.668 24c-3.896 0-6.934-3.039-6.934-6.974 0-1.851.701-3.487 1.811-4.714l-4.713 4.714-2.473-2.474 5.317-5.259c.72-.72 1.091-1.657 1.091-2.688V0h3.486v6.604c0 2.026-.72 3.74-2.123 5.143l-.156.156a6.945 6.945 0 0 1 4.693-1.812c3.954 0 6.973 3.136 6.973 6.935m-3.155.001c0-2.065-1.714-3.74-3.818-3.74s-3.778 1.753-3.778 3.74 1.674 3.779 3.778 3.779c2.103 0 3.818-1.773 3.818-3.779z"/></svg>
```

Comment:
Using https://www.beatport.com/images/beatport-logo-mobile.svg found in their mobile version header gives me the following differences:
![image](https://github.com/simple-icons/simple-icons/assets/12817388/7dc440a4-61a4-4965-a025-f85a4f229e0b)

Reply:
Think I'd used the Desktop site version. Will adjust to use mobile version!